internal/backend/test/tests.go

```diff
@@ -249,6 +249,19 @@ func (s *Suite) TestList(t *testing.T) {
 	b := s.open(t)
 	defer s.close(t, b)
 
+	// Check that the directory is empty to start with
+	var found []string
+	err := b.List(context.TODO(), restic.DataFile, func(fi restic.FileInfo) error {
+		found = append(found, fi.Name)
+		return nil
+	})
+	if err != nil {
+		t.Fatalf("List returned error %v", err)
+	}
+	if found != nil {
+		t.Fatalf("Repo not empty at start of test - contains: %v", found)
```

Comment:
`s/Repo/backend`

Reply:
I've fixed that now :-)  Haven't learnt all the restic words yet!

Reply:
Thanks!